fortuna/src/Login.js

```diff
@@ -0,0 +1,49 @@
+//@flow strict
+import * as React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import ReactDOM from 'react-dom';
+import MarketPlace from './MarketPlace.js';
+import './Login.css';
+  
+  //This is the Login Page Display
+  class LoginPageDisplay extends React.Component {
+    handleLoginClick = () => {
+        LoginFunction();
+      };
+    render(){
+      return (
+        <div id = "Parent">
+            <div class = "row">
+                <h4 class = "col-md-4">LeaderBoard</h4>
+                <h1 class = "col-md-4 text-center">Fortuna</h1>
+                <h4 class = "col-md-4 text-right">What is Fortuna?</h4>
+            </div>
+            <div class = "row styleForRow">
+              <div class="col text-center">
+                  <button type="button" class="btn btn-success btn-lg" onClick={this.handleLoginClick}>Login</button>
+              </div>
+            </div>
+            <div class = "row styleForRow">
+              <div class="col text-center">
+                  <button type="button" class="btn btn-secondary btn-sm">Signup</button>
+              </div>
+            </div>
+          </div>
+      );
+    }
+  
+  }
+
+  //This is the function that gets used when the player clicks login on the dashboard
+  function LoginFunction()
```

Comment:
This definitely needs to be fixed long term and is asking for stuff to break. We either need to use components' state/props to do this, or we need a flux store. I'm usually not a fan of merging stuff into master that is really bad practice, but I guess this stuff will probably be fixed when the login section gets handled, so it would make things easier probably.